packages/block-library/src/columns/edit.native.js

```diff
@@ -174,8 +174,9 @@ function ColumnsEditContainer( {
 
 	const getColumnsSliders = () => {
 		return innerColumns.map( ( column, index ) => {
-			const { valueUnit = '%' } =
+			const { valueUnit } =
 				getValueAndUnit( column.attributes.width ) || {};
```

Comment:
Hmmm, I'm wondering if there is any reason for returning `null` instead of `undefined` from `getValueAndUnit`. If it would be `undefined` then the default value would work properly :)